rig/place_and_route/place/hilbert.py

```diff
@@ -117,7 +117,8 @@ def place(vertices_resources, nets, machine, constraints):
             # Place the vertex
             unplaced_vertices.remove(constraint.vertex)
             placements[constraint.vertex] = loc
-        elif isinstance(constraint, ReserveResourceConstraint):
+        elif isinstance(constraint,  # pragma: no branch
```

Comment:
@mundya Hello from over the pond!

It isn't hard but it is not worth-while: The branch coverage in this case is complaining that it doesn't see any examples where there are constraints but the last one listed isn't a `ReserveResourceConstraint`. As such you could add a test with the constraints in an alternative order but it wouldn't really be worthwhile...